screens/Home/MyVcsTab.tsx

```diff
@@ -62,7 +62,7 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = (props) => {
   return (
     <React.Fragment>
       <Column fill style={{ display: props.isVisible ? 'flex' : 'none' }}>
-        <DownloadingIdPopUp />
+        <DownloadingVcPopUp />
```

Comment:
can we add the conmditional rendering here as well.

Reply:
I have already rendered it by using conditional rendering in the above code with the "**isVisible**" prop state. Also, It working fine as per conditions.

Reply:
isVisible field only toggles the display of the component. although this will get rendered all the time. since we want to render the component only when hasPermission is false, so its better to use conditional rendering like this.

{isVisible &&  < Component / >}